src/content/workflow/rebase.mdx

```diff
@@ -0,0 +1,126 @@
+
+Rebasing is a method to integrate changes from one branch into another. It's
+essentially like "replaying" changes from your branch onto another branch. This
+is helpful when you want to ensure your changes are built upon the most recent
+commits of the branch you're targeting.
+
+Rebasing a pull request in GitHub allows you to update the entire commit
+history of a branch onto its base branch, while the base branch received new
+commits.
+
+Mergify allows to rebase pull requests in two different ways:
+
+- Either by using a the [`rebase` command](/commands/rebase) which rebases the
+  pull request immediately;
+
+- Either by using an automation rule that automatically triggers a rebase
+  when a pull request meets conditions.
+
+## Rebasing a PR from GitHub
+
+To instantly rebase a pull request without leaving GitHub, uses the Mergify
+rebase command by typing:
+
+```text
+@mergifyio rebase
+```
+
+<Screenshot src={rebaseScreenshot} />
+
+This command triggers a `git rebase` operation and force-push the rebased
+branch.
+
+import RebaseAdvantage from '../commands/_rebase_advantages.mdx';
+
+<RebaseAdvantage />
+
+## Automatically Rebasing PRs
+
+You might be tempted to rebase pull request automatically when they are behind
+their base branch, such as `main` or `master`. The [`rebase`
+action](/workflow/actions/rebase) is a great candidate to help in such a case.
+
+You can write an automation rule that does exactly this for you:
+
+```yaml
+pull_request_rules:
+  - name: continuously rebase pull request when it's labeled with `rebase`
+    conditions:
+      - label=rebase
+    actions:
+      rebase:
+```
+
+Note that this rebases the pull request as long as it has the label.
+
+If you wanted to rebase only once, you would also need to remove the label once
+the rebase is done:
+
+```yaml
+pull_request_rules:
+  - name: rebase pull request once when it's labeled with `rebase`
+    conditions:
+      - label=rebase
+    actions:
+      rebase: {}
+      label:
+        remove:
+          - rebase
+```
+
+### Rebasing Outdated Pull Requests
+
+As rebasing a PR is likely to re-trigger your CI, rebasing your PR continuously
+could be time consuming for your CI system. Rather than doing it on every merge
+done in the base branch, you can rebase only when the PR is behind a number of
+commits. For example:
+
+```yaml
+pull_request_rules:
+  - name: rebase pull request when it's more than 10 commits behind main
+    conditions:
+      - base=main
+      - "#commits-behind>=10"
+    actions:
+      rebase:
+```
+
+With this rule, as soon as the pull request is more than 10 commits behind the
+`main` branch, Mergify automatically rebases the pull request which therefore
+retriggers the CI. This is useful to make sure that you don't merge pull
+request that are too much out-of-date.
+
+<Warning>
+  If your goal is to only merge pull request that are rebased and retested
+  before being merged, you are actually looking for a [merge
```

Comment:
```suggestion
  If your goal is to only merge pull request that are rebased and retested, you are actually looking for a [merge
```
WDYT ?

Reply:
I think I prefer the explicit version :)